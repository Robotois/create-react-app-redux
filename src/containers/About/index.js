import React from 'react';

export default class About extends React.PureComponent {
  render() {
    return (
      <div>
        <h1>About</h1>
        <p>we all are pros!</p>
      </div>
    );
  }
}
