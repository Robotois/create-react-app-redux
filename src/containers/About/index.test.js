import React from 'react';
import renderer from 'react-test-renderer';

import About from './';

it('renders without crashing', () => {
  const component = renderer.create(<About />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
