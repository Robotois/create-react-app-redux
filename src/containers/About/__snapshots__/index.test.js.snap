// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div>
  <h1>
    About
  </h1>
  <p>
    we all are pros!
  </p>
</div>
`;
